masterlist.yaml

```
@@ -18725,69 +18849,35 @@ plugins:
-      - <<: *patch3rdParty
-        subs:
-          - 'Cloaks of Skyrim'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Cloaks.esp") and not active("JKs Temple of Talos - Cloaks of Skyrim patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Enhanced Lights and FX'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("EnhancedLightsandFX.esp") and not active("JKs Temple of Talos - ELFX Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Interesting NPCs'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("3DNPC.esp") and not active("JKs Temple of Talos - 3DNPC patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Oblivion Artifact Pack'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("WZOblivionArtifacts.esp") and not active("JKs Temple of Talos - Oblivion Artifacts patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Palaces and Castles Enhanced'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Palaces Castles Enhanced.esp") and not active("JKs Temple of Talos - PCE patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Snazzy Furniture and Clutter Overhaul (SFCO)'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Snazzy Furniture and Clutter Overhaul.esp") and not active("JKs Temple of Talos - SFCO patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Solitude and Temple Frescoes 2019'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("SolitudeTempleFrescoes.esp") and not active("JKs Temple of Talos - Solitude and Temple Frescoes Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'The Elder Scrolls Online Imports'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("ESOimports.esp") and not active("JKs Temple of Talos - ESO Imports Patch.esp")'
-      - <<: *patch3rdParty
-        subs:
-          - 'Unofficial Skyrim Special Edition Patch'
-          - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
-        condition: 'active("Unofficial Skyrim Special Edition Patch.esp") and not active("JKs Temple of Talos - USSEP patch.esp")'
+      - <<: *compatPatch_JKIPC
+        subs: [ '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)' ]
```

@cecell, sorry, I didn't realize there was another message there. Only the following change is actually necessary.

```suggestion
      - *compatPatch_JKIPC
```